build.sh

```diff
@@ -3,51 +3,8 @@
 # Copyright (c) .NET Foundation and contributors. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
-
-# Set OFFLINE environment variable to build offline
-
 set -e
```

Comment:
> I guess I was assuming set -e put it into offline mode because I don't know that that does.

Ahh!  What `set -e` means is "if any command in the script file fails, exit immediately instead of continuing."

A nice hygienic thing to stop things early if something you expected would work fails.